assets/js/src/newsletter_editor/components/sidebar.js

```diff
@@ -327,7 +327,38 @@ Module.SidebarPreviewView = Marionette.View.extend({
       }).fail(function (response) {
         if (response.errors.length > 0) {
           MailPoet.Notice.error(
-            response.errors.map(function (error) { return error.message; }),
+            response.errors.map(function (error) {
+              let errorMessage = `
+                <p>
+                  ${MailPoet.I18n.t('newsletterPreviewErrorNotice').replace('%$1s', window.config.mtaMethod)}:
+                  <i>${error.message}</i>
+                </p>
+              `;
+
+              if (window.config.mtaMethod === 'PHPMail') {
```

Comment:
Using current page's `window.config.mtaMethod` has a small issue in that if you change a sending method in another tab while working in an editor, it will format the error message according to the old type. Perhaps not critical, but might confuse someone someday.

Reply:
Right, I was looking into that but unfortunately `ErrorResponse doesn't have that info and it would be quite a lot of work to include it everywhere. In the Preview it's not such a big issue, however, I'm afraid the same problem happens [here](https://github.com/mailpoet/mailpoet/blob/dfda8296d70af2463034aac04ffd5820fdc00016/assets/js/src/newsletters/listings/mixins.jsx#L420), it's just not visible. The thing is there is no sending method in the `mta_log` in DB which means the currently used method is probably appended somewhere on the way. In this case it could really be confusing sometimes.

Reply:
Let's leave it for now but keep it in mind if something can happen, it will happen given enough users. So we might go back and put extra work into eliminating these inconsistencies in the future.